### ISSUE TITLE
Make option to hide alerts more obvious

### ISSUE THREAD

**Issue by barbeau** [OneBusAway/onebusaway-android#1041]
**Summary:** 

I've gotten 3-4 user complaints over the last two weeks saying that the alerts are blocking the list of arrival times. Agencies are publishing tons of alerts due to dynamic COVID-19 operations.

You can tap on an arrival and then select "Hide" to collapse it, and users are happy with this once they know about it. But apparently it's not easy to discover. Maybe a tutorial screen on this would help that shows for everyone (we have a similar prompt on the trip plan screen).

**Steps to reproduce:** 

1. Open any region with a lot of alerts for stops
1. Tap on the stop

**Expected behavior:** 

Make it obvious that I can hide alerts

**Observed behavior:** 

Users aren't discovering the ability to hide alerts.

**Comment by barbeau** [comment]
Another comment from a user:

>I'm a low-vision transit rider and I use large fonts on my phone to be able to reach the text.  
With all the service change alerts in the One Bus Away system, I'm currently unable to see more than the first or second bus scheduled to come at any given stop, which is a challenge for stops downtown that serve multiple bus lines. I can't see my bus on the schedule at all, and there appears to be no way to close the service alerts so I could see more of the scheduled busses.

**Comment by bagriffith** [comment]
Agreed, I didn't know you could hide them until I found this. My stop has enough notifications that you can't see all of the routes.

![Screenshot_20220118-161635](https://user-images.githubusercontent.com/12106328/150039908-4bb3fe6a-ab6e-4551-b034-a1e81de49421.png)

**Comment by barbeau** [comment]
Yes, this has become an issue again in the last few weeks due to increase in service cancellations from the COVID-19 spike.

Ideally most of these alerts would instead be communicated as trip-level cancellations to the app so they could be displayed inline with other arrivals. I implemented most of the code for this in this PR:
https://github.com/OneBusAway/onebusaway-android/pull/1034

![79491263-3071df80-7fec-11ea-93da-a6723b312b00](https://user-images.githubusercontent.com/928045/150053626-96105a55-c9b9-4ce0-ba74-8ab8b6c64a4a.png)

But it still requires some server side support that's not implemented yet. That would fall on Sound Transit's side of things and I've reached out to them to see if we can make it happen.
